shell/run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
is this file normal? :)

Reply:
I've informed @lenadoc, and asked if she is OK with this. 

> Wow, thank you very much for the shell script! 

It makes running the command easier. I don't like to type that command manually each time or copy-paste it. 😺